.github/workflows/auto-project.yml

```
@@ -0,0 +1,22 @@
+name: Auto Project
+on:
+  issues:
+    types: [opened]
+  pull_request:
+    types: [opened]
+
+jobs:
+  add_to_project::
```

actionlint (1.7.4)

invalid job ID "add_to_project:". job ID must start with a letter or _ and contain only alphanumeric characters, -, or _

(id)